### PR TITLE
Farmboy. 

### PR DESCRIPTION
**Pull request by sweetdream0729**
For #72

### CODE REVIEW

File: locations/spiders/farmboy.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    def parse(self, response):
+
+        stores = response.xpath('//div[@id="portfolio"]/article')
+        for store in stores:
+            properties = {
+                "ref": store.xpath('div/h3/text()').extract_first(),
+                "name": store.xpath('div/h3/text()').extract_first(),
+                "addr_full": store.xpath('div/div/p/text()').extract_first(),
+                "postcode": self.postCode(store.xpath('div/div/p/text()[last() -1 ]').extract_first()),
```
Comment (iandees) [inline locations/spiders/farmboy.py]: Looks like the province is extracted as the postcode sometimes.

File: locations/spiders/farmboy.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                "phone": self.phone(store.xpath('div/div/div[@id="cinfo"]/p/text()').extract_first()),
+                "opening_hours": self.parse_hours(store.xpath('div/div/div[@id="sinfo"]/table[1]/tbody/tr'))
+            }
+
+            yield GeojsonPointItem(**properties)
+
+    def city(self, data):
+        str_list = data.split(',')
+        print(str_list[0])
```
Comment (iandees) [inline locations/spiders/farmboy.py]: Remove debug `print()`

File: locations/spiders/farmboy.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        return "; ".join(opening_hours)
+
+    def parse(self, response):
+
+        stores = response.xpath('//div[@id="portfolio"]/article')
+        for store in stores:
+            properties = {
+                "ref": store.xpath('div/h3/text()').extract_first(),
+                "name": store.xpath('div/h3/text()').extract_first(),
+                "addr_full": store.xpath('div/div/p/text()').extract_first(),
```
Comment (iandees) [inline locations/spiders/farmboy.py]: A couple of the results only have ref/name. Can you double-check the xpaths?

**Comment by sweetdream0729** [comment]
Sorry, for missing above issues.
There is a store coming soon.
Let me know if there is any other issue.
Thanks.

**Comment by sweetdream0729** [comment]
Can I add new stores to work?

**Pull request merged**